Shared/Samples/Trace utility network/TraceUtilityNetworkView.Model.swift

```diff
@@ -201,7 +205,9 @@ extension TraceUtilityNetworkView {
                 try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
                 try await network?.load()
             } catch {
-                updateUserHint(withMessage: "An error occurred while loading the network.")
+                Task {
+                    await updateUserHint(withMessage: "An error occurred while loading the network.")
+                }
                 return
```

Comment:
This method is already marked as async so you don't need a task here.

Reply:
Oops, good catch, thank you